robot/Cumulus/resources/GiftEntryPageObject.py

```diff
@@ -133,16 +136,37 @@ def select_object_group_field(self,object_group,field):
             time.sleep(2)       
         field_checkbox=npsp_lex_locators["gift_entry"]["field_input"].format(field,"input")  
         check=self.selenium.get_webelement(field_checkbox)
-        if not check.is_selected():
-            try:
-                self.salesforce._jsclick(field_checkbox)
-            except ElementClickInterceptedException:
-                self.selenium.execute_javascript("window.scrollBy(0,0)")
-                self.salesforce._jsclick(field_checkbox)
-        label=": "+field    
-        self.selenium.wait_until_page_contains(label)
+        if action.lower()=='select':
+            if not check.is_selected():
+                try:
+                    self.salesforce._jsclick(field_checkbox)
+                except ElementClickInterceptedException:
+                    self.selenium.execute_javascript("window.scrollBy(0,0)")
+                    self.salesforce._jsclick(field_checkbox)
+            label=": "+field    
+            self.selenium.wait_until_page_contains(label)
```

Comment:
Why are you specifically checking for `": <field>"`. Does checking for `"<field>"` return more than 1 result ?

Reply:
yes, this page has field names shared across multiple components(for a reason) so I have to do it this way